about.html

```diff
@@ -0,0 +1,128 @@
+                    <p class="text-p about-text-p p-3 border-secondary bg-white">One good thing about music, when it hits you feel no pain.</p>
+                </div>
+                <div class="text-[#272a31]">
+                    <h2 class="text-[25px] md:text-[40px] text-[#272a31]">To know more about this upcoming festival, contact me.</h2>
+                    <b class="under">kgomotsonacane@gmail.com</b>
+                </div>
+            </div>
+        </section>
+        <section class="program-about">
+            <br>
+            <h4 class="text-center section-title-black">Music Festival<br>On Winter</h4>
+            <hr class="line-name">
+            <div class="program-container"></div>
+            <br>
+            <p>This was intended to help people from different ages,<br>religions, and cultures to get together and enjoy themselves</p><br>
+            <div class="first-container"><img src="assets/trebleClef.png" class="main-logo" width="90%" alt="logo"></div>
+        </section><hr>
+        <section class="past-program bg-white">
+            <div class="first-container">
+                <br>
+                <h4 class="text-center section-title-black">See the past Music Festivals</h4>
+                <hr class="line-name">
+                <p>Explore this in the name of fun and happiness, I dare you</p>
+                <div class="past-summit">
+                    <div class="summit-card summit-one text-center mt-3">
+                        <div class="summit-sub">
+                            <h2 class="text-white">2016</h2>
+                            <p class="text-white"> Gauteng Concert</p>
+                        </div>
+                    </div>
+                    <div class="summit-card summit-two text-center mt-3">
+                        <div class="summit-sub">
+                            <h2 class="text-white">2016</h2>
+                            <p class="text-white"> Gauteng Concert</p>
+                        </div>
+                    </div>
+                </div>
+                <br><br>
+            </div>
+        </section>
+        <section class="more-partners more-partners-about bg-secondary">
+            <div class="first-container"><br>
+                <h4 class="text-center section-title-white">Partner</h4><hr class="line-name">
+                <div class="first-container p-div text-center">
+                    <div class="first-group">
+                        <h4 class="first-p">Spotify</h4>
+                        <h4 class="second-p">YouTube</h4>
+                        <h4 class="third-p">DStv</h4>
+                        <h4 class="fourth-p">Amazon</h4>
+                        <h4 class="fifth-p">TikTok</h4>
+                    </div>
+                </div>
+            </div><br><br>
+        </section>
+    </main>
+    <footer class="bottom-last bg-light" id="updates">
+        <div class="first-container bottom-part">
```

Comment:
- [ ] Just as I pointed out above, This section of your about me page also overlaps each other.
- - You can see the menu is this section which is not meant to be.
- - Also the spacing and margin by the left is big compared to the one in the given desgin
- - The footer text tends not to be vivble here, I'll suggest you update this accordingly

- Screenshot:
![Selection_046](https://github.com/Kgomotso196/FirstCapstoneProject/assets/107632780/fbd67f09-0c4f-4b94-9ddf-8c6c069f4c9f)